src/test/java/apoc/load/XmlTest.java

```diff
@@ -71,7 +79,9 @@ public void testMixedContent() {
         testCall(db, "CALL apoc.load.xml('file:src/test/resources/mixedcontent.xml')", //  YIELD value RETURN value
                 (row) -> {
                     Object value = row.get("value");
-                    assertEquals("{_type=root, _children=[{_type=text, _children=[text0, {_type=mixed}, text1]}, {_type=text, _text=text as cdata}]}", value.toString());
+                    //assertEquals("{_type=root, _children=[{_type=text, _children=[text0, {_type=mixed}, text1]}, {_type=text, _text=text as cdata}]}", value.toString());
```

Comment:
@sarmbruster we changed the test here because we are using Node objects to create the final JSON representation in order to support both XPath path or not.
With this change we can't guarantee the order of a mixed content as it is in the XML document.
For example in this test before our changes there was `[text0, {_type=mixed}, text1]` while now is `[{_type=mixed}, text0, text1]`. I'm also wondering if `[{_type=mixed}, text0, text1]` is the correct JSON structure or maybe it should be `[{_type=mixed}, {_text=[text0, text1]}]`.

Thanks